packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -231,8 +228,16 @@ export default {
       animationStart: null,
       animationLong: null,
       animationDuration: 300,
+      isVisibleOnMobile: false,
```

Comment:
what is the different between `isNavVisible` and `isVisibleOnMobile`?

Reply:
Maybe it is strange idea, but I added isVisibleOnMobile to inject mobile observer into internal components. And isNavVisible is just the  prop  to set visibility of the nav.

Reply:
Is that make sense? Or maybe there should be separeted mobile observer imported to both internal components?

Reply:
I'll remove isVisibleOnMobile.